server/services/GoService.js

```diff
@@ -121,11 +121,14 @@ export default class GoService {
         });
     };
     // Refresh pipeline names and poll every day for new
-    if (conf.groupPipelines) {
-      getPipelineGroups();
-    }
-    refreshPipelinesAndPollForUpdates();
-    setInterval(refreshPipelinesAndPollForUpdates, this.checkPipelinesInterval);
+    const fullRefresh = () => {
+      Logger.info('Performing full refresh');
```

Comment:
Not sure this logging is needed, maybe use `Logger.debug` instead?

Reply:
ok, will do

Reply:
ok. I have change it, but is seems that the change is having hard time to propagate to this pull request...